packages/transition-frontend/ui-tests/testHelpers.ts

```diff
@@ -0,0 +1,226 @@
+ * @param {LoginMethods} options.loginMethod - The method used to log in. By default, presses the login button.
+ */
+export const loginTest: LoginTest = ({ context, loginMethod }) => {
+    test(`Login to the test account with method ${loginMethod} - ${getTestCounter(context, `${loginMethod}`)}`, async () => {
+        const userNameField = context.page.locator("id=usernameOrEmail");
+        await userNameField.fill(process.env.PLAYWRIGHT_TEST_USER as string);
+        const passwordField = context.page.locator("id=password");
+        await passwordField.fill(process.env.PLAYWRIGHT_TEST_PASSWORD as string);
+        switch(loginMethod) {
+            case 'clickLoginButton':
+                const loginButton = context.page.getByRole('button', { name: 'Login' });
+                await loginButton.click();
+                break;
+            case 'enterOnUsernameField':
+                await userNameField.click();
+                await context.page.keyboard.press('Enter');
+                break;
+            case 'enterOnPasswordField':
+                await passwordField.click();
+                await context.page.keyboard.press('Enter');
+                break;
+        }
+        const logoutButton = context.page.getByRole('button', { name: 'Logout' });
+        await expect(logoutButton).toBeVisible();
+        const userButton = context.page.getByRole('button', { name: process.env.PLAYWRIGHT_TEST_USER });
+        await expect(userButton).toBeVisible();
+    });
+};
+
+/**
+ * Logout, and verify we end up back on the login page.
+ * @param {Object} options - The options for the test.
+ */
+export const logoutTest: LogoutTest = ({ context }) => {
+    test(`Logout from survey - ${getTestCounter(context, '')}`, async () => {
+        const logoutButton = context.page.getByRole('button', { name: 'Logout' });
+        await logoutButton.click();
+        await expect(context.page).toHaveURL('/login');
+    });
+};
+
+/**
+ * Click on one of the sections on the left menu, and check that the right panel is the correct one.
+ * @param {Object} options - The options for the test.
+ * @param {string} options.section - The section we click.
+ */
+export const clickLeftMenuTest: LeftMenuTest = ({ context, section }) => {
+    test(`Click the ${section} section of the left menu - ${getTestCounter(context, `${section}`)}`, async () => {
+        //test.setTimeout(60000);
+        const leftMenu = context.page.locator("//nav[@id='tr__left-menu']/ul[@class='tr__left-menu-container']");
+        const sectionButton = leftMenu.locator(`//li/button[@data-section='${section}']/span/img`);
+        await sectionButton.click();
+        const rightPanel = context.page.locator("//section[@id='tr__right-panel']/div[@class='tr__right-panel-inner']");
+        const rightPanelTitle = rightPanel.locator("//h3").nth(0);
+        let expectedRightPanelTitle;
+        switch(section) {
+            case 'agencies':
```

Comment:
This part seems a bit hard to maintain as it depends on translation and configuration. Not fully sure if there's a value on checking the whole list.